fabtests/benchmarks/dgram_pingpong.c

```diff
@@ -49,6 +49,9 @@ static int run(void)
 	if (ret)
 		return ret;
 
+	if (ft_check_opts(FT_OPT_FORK_CHILD))
+		ft_fork_child();
```

Comment:
We need to place this somewhere else, so we aren't duplicating it in every single test.